include/exadg/structure/spatial_discretization/operator.cpp

```diff
@@ -726,12 +735,18 @@ Operator<dim, Number>::compute_initial_acceleration(VectorType &       accelerat
     {
       body_force_operator.evaluate_add(rhs, displacement, time);
     }
+
+    // revert scaling factor to initialized value
+    elasticity_operator_nonlinear.set_scaling_factor_mass_operator(scaling_factor_mass);
```

Comment:
I think one could even shift this line even before evaluation the body force operator. This would make the "manipulation" more local in the code.

Reply:
Ok, now it is as close as possible - even though it fit on one screen before ; )